public/app/plugins/datasource/grafana/datasource.ts

```diff
@@ -177,7 +191,17 @@ export class GrafanaDatasource extends DataSourceWithBackend<GrafanaQuery> {
   }
 
   async getAnnotations(options: AnnotationQueryRequest<GrafanaQuery>): Promise<DataQueryResponse> {
-    const templateSrv = getTemplateSrv();
+    const query = options.annotation.target as GrafanaQuery;
+    if (query?.queryType === GrafanaQueryType.TimeRegions) {
+      const frame = doTimeRegionQuery(
+        options.annotation.name,
+        query.timeRegion!,
+        options.range,
+        getDashboardSrv().getCurrent()?.timezone
```

Comment:
```suggestion
        options.timezone
```
timezone should be in the reqeust